modules/ezcontent_node/modules/ezcontent_article/ezcontent_article.install

```diff
@@ -32,3 +34,18 @@ function ezcontent_article_update_8002() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Configuration update.
+ */
+function ezcontent_article_update_8003() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+  $module_path = drupal_get_path('module', 'ezcontent_article');
+  $file = 'entityqueue.entity_queue.featured_articles';
```

Comment:
@arshadkhan53 We would need to install the entityqueue module programmatically before importing its config.